### PR TITLE
Czech Localization

### PR DESCRIPTION
**Pull request by tomik148**

### CODE REVIEW

**Comment by BeefEX** [comment]
@tomik148 to sem chtel delat ja! :D

File: Assets/StreamingAssets/Localization/cs_CZ.lang [hunk 1]
```diff
@@ -0,0 +1,25 @@
+﻿build_floor=Postav podlahu
+remove_floor=Odstraň podlahu
+deconstruct_furniture=Rozmontovat nábitek
```
Comment (BeefEX) [inline Assets/StreamingAssets/Localization/cs_CZ.lang]: Wrong spelling

Comment (tomik148) [inline Assets/StreamingAssets/Localization/cs_CZ.lang]: can i edit this pull of i have to close this one and make another one?

Comment (BeefEX) [inline Assets/StreamingAssets/Localization/cs_CZ.lang]: you can commit changes to this one

Comment (tomik148) [inline Assets/StreamingAssets/Localization/cs_CZ.lang]: and how exactly

Comment (BeefEX) [inline Assets/StreamingAssets/Localization/cs_CZ.lang]: i dont know, i am new to git too :(

Comment (SanteriHetekivi) [inline Assets/StreamingAssets/Localization/cs_CZ.lang]: Just make the changes, commit and push.

**Comment by Tranberry** [comment]
I would like your input on #153

**Comment by SanteriHetekivi** [comment]
Could you add translations to Czech for:
furn_power_generator=Power Generator
furn_power_generator_desc=Generates power out of nothing.

They are missing in current master, but pull request #169 should add them in.

**Comment by tomik148** [comment]
Sure

**Comment by vogonistic** [comment]
@tomik148 This PR conflicts with the auto-generated translation that is already merged. Can you update it so it merges cleanly? Please replace the auto-generated translation with yours.

**Comment by tomik148** [comment]
@vogonistic resolved

**Pull request merged**